resources/idlharness.js

```diff
@@ -2160,7 +2176,17 @@ IdlInterface.prototype.test_immutable_prototype = function(type, obj)
             } catch (err) {}
         });
 
-        assert_throws(new TypeError(), function() {
+        var setter;
```

Comment:
Can we use modern `let` (or `const`) rather than `var` (unless they have to be global for some reason?)

Reply:
We can use `let`, sure.  At one point we were avoiding that in wpt due to poor support, but I think it's good to put that behind us.  ;)